apps/web-giddh/src/app/home/components/total-overdues/total-overdues-chart.component.html

```diff
@@ -25,15 +25,14 @@
 				</div>
 			</div>
 			<div class="chart-div" google-chart chart="chartData" *ngIf="!requestInFlight">
-				<perfect-scrollbar [scrollIndicators]="true">
 					<chart [options]="totaloverDueChart" *ngIf="dataFound" #currentRatio></chart>
 					<div class="noDataAvailable" *ngIf="!dataFound">
 						<img src="{{imgPath}}noDataChart.svg" alt="no data found">
 					</div>
 					<div class="clearfix mrB2">
 						<div class="">
```

Comment:
Blank div

Reply:
Is blank class is in green color or red?